### ISSUE TITLE
Install instructions

### ISSUE THREAD

**Issue by cloud9business** [laracasts/Presenter#32]
I think the installation instructions for the Laravel 5 compatible versions need to change to reference 0.2.* 
"laracasts/presenter": "0.2.*"

**Comment by subdesign** [comment]
yes, with 0.1.* error dropped, with 0.2.* (currently 0.2.1) installed

**Comment by aoamusat** [comment]
Does changing the version to "0.2.*" works for Laravel 5.x?? @subdesign
